src/Uno.Extensions.Localization.UI/LocalizationService.cs

```diff
@@ -3,6 +3,10 @@
 
 namespace Uno.Extensions.Localization;
 
+/// <summary>
+/// An implementation of <see cref="ILocalizationService"/> that uses the <see cref="LocalizationConfiguration"/>
+/// to determine the supported cultures and allow for changing the current culture.
+/// </summary>
 public class LocalizationService : IServiceInitialize, ILocalizationService, IDisposable
```

Comment:
Can this be marked as internal?

Reply:
Addressed with 75f9665a96b59f6b97c160a99450674ab620a73a

Reply:
Include the commit in this PR